examples/basic/README.md

```diff
@@ -0,0 +1,11 @@
+# Basic example
+
+An example that creates the following infrastructure:
+
+ - Create a new resource group if one is not passed in.
+ - Create a new Hyper Protect Crypto Services instance.
+
+**Note:** This example doesn't initialize the service instance. There are multiple ways to initialize the service instance few of them include some manual steps, they are as follows:
+ - [Initializing service instances by using smart cards and the Hyper Protect Crypto Services Management Utilities](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-initialize-hsm-management-utilities) : This approach gives you the highest security, which enables you to store and manage master key parts using smart cards.
```

Comment:
I would move those details up to the root README.md rather than burrying them in the example

Reply:
It is already there in README as well. We have mentioned at both places.

Reply:
perhaps just link to it then so that we do not have duplicate text to update in the future. @maheshwarishikha

Reply:
make the changes. 
Actual content is in README now and example's readme refers to main readme.